mediaire_toolbox/data_cleaner.py

```diff
@@ -75,9 +132,22 @@ def clean_up(self, dry_run=False):
                     "Current total size is too big (%s bytes), need to delete some data to free up space" % current_size)
             if delete:
                 removed.append(folder)
+                pre_clean_size = self.current_size(folder)
                 if dry_run:
```

Comment:
This if / else with the dry_run is not clear to me... some code repetition that isn't necessary probably. After you fix the issue of passing the parameter to the child method you can check it out.